mne/channels/montage.py

```diff
@@ -1270,3 +1356,221 @@ def _set_montage(info, montage, update_ch_names=False, set_dig=True):
+        considered as EEG channels.
+    unit : 'm' | 'cm' | 'mm'
+        Unit of the digitizer file. Polhemus ISOTrak systems data is usually
+        exported in meters. Defaults to 'm'
+
+    Returns
+    -------
+    montage : instance of DigMontage
+        The montage.
+
+    See Also
+    --------
+    make_dig_montage
+    read_polhemus_fastscan
+    """
+    VALID_FILE_EXT = ('.hsp', '.elp', '.eeg')
+    VALID_SCALES = dict(mm=1e-3, cm=1e-2, m=1)
+    _scale = _check_unit_and_get_scaling(unit, VALID_SCALES)
+
+    _, ext = op.splitext(fname)
+    _check_option('fname', ext, VALID_FILE_EXT)
+
+    if ext == '.elp':
+        data = _read_isotrak_elp_points(fname)
+    else:
+        # Default case we read points as hsp since is the most likely scenario
+        data = _read_isotrak_hsp_points(fname)
+
+    if _scale != 1:
+        data = {key: val * _scale for key, val in data.items()}
+    else:
+        pass  # noqa
+
+    if ch_names is None:
+        keyword = 'hpi' if ext == '.elp' else 'hsp'
+        data[keyword] = data.pop('points')
+
+    else:
+        points = data.pop('points')
+        if points.shape[0] == len(ch_names):
+            data['ch_pos'] = dict(zip(ch_names, points))
+        else:
+            raise ValueError((
+                "Length of ``ch_names`` does not match the number of points"
+                " in {fname}. Expected ``ch_names`` length {n_points:d},"
+                " given {n_chnames:d}"
+            ).format(
+                fname=fname, n_points=points.shape[0], n_chnames=len(ch_names)
+            ))
+
+    return make_dig_montage(**data)
+
+
+def _get_polhemus_fastscan_header(fname):
+    with open(fname, 'r') as fid:
+        header = [l for l in takewhile(lambda line: line.startswith('%'), fid)]
+
+    return ''.join(header)
+
+
+def read_polhemus_fastscan(fname, ch_names=None, unit='mm'):
+    """Read Polhemus FastSCAN digitizer data from a ``.txt`` file.
+
+    Parameters
+    ----------
+    fname : str
+        The filepath of .txt Polhemus FastSCAN file.
+    ch_names : None | list of str
+        The names of the points. This will make the points
+        considered as EEG channels.
+    unit : 'm' | 'cm' | 'mm'
+        Unit of the digitizer file. Polhemus FastSCAN systems data is usually
+        exported in millimeters. Defaults to 'mm'
+
+    Returns
+    -------
+    points : array, shape (n_points, 3)
+        The digitization points in digitizer coordinates.
+
+    See Also
+    --------
+    read_dig_polhemus_isotrak
+    make_dig_montage
+    """
+    VALID_FILE_EXT = ['.txt']
+    VALID_SCALES = dict(mm=1e-3, cm=1e-2, m=1)
+    _scale = _check_unit_and_get_scaling(unit, VALID_SCALES)
+
+    _, ext = op.splitext(fname)
+    _check_option('fname', ext, VALID_FILE_EXT)
+
+    if _get_polhemus_fastscan_header(fname).find('FastSCAN') == -1:
+        raise ValueError(
+            "%s does not contain Polhemus FastSCAN header" % fname
+        )
+
+    if _scale == 1:
```

Comment:
you could simplify this by having removing the if statement. if it is equal to 1, it will just return itself